ckan/requirements-dev.txt

```diff
@@ -0,0 +1,139 @@
+alabaster==0.7.12
```

Comment:
This isn't the typical dev requirements file. It's all the normal requirements _plus_ the dev requirements. `poetry` doesn't have a simple way to create separate files. You can either generate the non-dev requirements or _all_ requirements. I'm sure we could write some sort of automated script to fix this, but since we're not deploying from here, I figured we just use the dev file in our Dockerfile, and Tom can use the normal file for security checks, etc.